core/src/leader_slot_banking_stage_metrics.rs

```diff
@@ -300,6 +307,37 @@ impl LeaderSlotMetrics {
     }
 }
 
+// Metrics describing vote tx packets that were processed in the tpu vote thread as well as
+// extraneous votes that were filtered out
+#[derive(Debug, Default)]
+pub(crate) struct VotePacketCountMetrics {
+    // How many votes ingested from gossip were dropped
+    dropped_gossip_votes: u64,
+
+    // How many votes ingested from tpu were dropped
+    dropped_tpu_votes: u64,
+}
+
+impl VotePacketCountMetrics {
+    fn new() -> Self {
+        Self { ..Self::default() }
+    }
+
+    fn report(&self, id: u32, slot: Slot) {
```

Comment:
I just learned this recently: the `as i64` in this function are useless. When the macro expands it already adds an `as i64` because of the 3rd tuple element

Reply:
this is good to know!